MapleServer2/PacketHandlers/Game/FieldEnterHandler.cs

```diff
@@ -83,7 +83,7 @@ public override void Handle(GameSession session, PacketReader packet)
 
         if (player.Party is not null)
         {
-            session.Send(PartyPacket.UpdatePlayer(player));
+            player.Party.BroadcastPacketParty(PartyPacket.UpdatePlayer(player));
         }
```

Comment:
couldn't we just do `player.Party?.BroadcastPacketParty(PartyPacket.UpdatePlayer(player));`